_pages/index.md

```diff
@@ -19,11 +19,21 @@ hero:
       Three people, two with visible disabilities, relaxing and talking outside.
     mobile: |-
       Five people, some with visible disabilities, hanging out on a rooftop deck while talking and laughing.
-  content: |-
-    Disability rights are civil rights. From voting to parking, the ADA is a law that protects people with disabilities in many areas of public life.
   heading: |-
     The Americans with Disabilities Act (ADA) protects people with disabilities from discrimination.
 
+# 'content' is a section of text that displays underneath the heading.
+  # content: |-
+  #   Disability rights are civil rights. From voting to parking, the ADA is a law that protects people with disabilities in many areas of public life.
+
+# 'news-widget' is an information box that shows on underneath the heading, and underneath content if it is populated. It is used to highlight and link to a particular news item.
```

Comment:
this documentation can go in the wiki: https://github.com/usdoj-crt/beta-ada/wiki